democracy/tests/unittest/test_user_data_remover.py

```diff
@@ -0,0 +1,358 @@
+from datetime import datetime
+
+import freezegun
+import pytest
+import reversion
+from django.utils import timezone
+from reversion.models import Version
+
+from democracy.factories.hearing import (
+    CommentImageFactory,
+    HearingFactory,
+    LabelFactory,
+    MinimalHearingFactory,
+    SectionCommentFactory,
+    SectionFactory,
+    SectionFileFactory,
+    SectionImageFactory,
+)
+from democracy.factories.organization import OrganizationFactory
+from democracy.factories.poll import SectionPollFactory, SectionPollOptionFactory
+from democracy.models import (
+    ContactPerson,
+    Project,
+    ProjectPhase,
+    SectionPollAnswer,
+    SectionType,
+)
+from democracy.utils import user_data_remover
+from kerrokantasi.models import User
+from kerrokantasi.tests.factories import UserFactory
+
+
+@pytest.mark.django_db
+class RemoveUserTestsBase:
+    @pytest.fixture(autouse=True)
+    def init_test_data(self):
+        self.threshold_time = timezone.now() - timezone.timedelta(days=30)
+        self.user = UserFactory(username="user")
```

Comment:
Imo the variable / fixture naming matters - where it is initialized don't. Since if you want to know what the fixture has been eaten you still need to travel your IDE to the point of creation. But what comes to base class that's a bit overkill here (pros for defining variables and annotations once but that a loose pro and cons it clearly can obfuscate things).
As conclusion I'm agreeing you with my comments in mind and I'll remove the base class and change these vars to fixtures because this case it seems clearer 👍